module_a/src/main/java/org/sonarqube/A.java

```diff
@@ -4,9 +4,9 @@
   String message = "foo";
 
   public String foo() {
+// FIXME
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Take the required action to fix the issue indicated by this comment. ([squid:S1134](http://localhost:9000/coding_rules#rule_key=squid%3AS1134))

[See it in SonarCloud](http://localhost:9000/project/issues?id=org.sonarqube%3Apr-decoration&pullRequest=41&issues=AWKWSYTrRVvknov3mI8m&open=AWKWSYTrRVvknov3mI8m)

[ISSUE_KEY]: AWKWSYTrRVvknov3mI8m